### PR TITLE
Feature/day6 selenium1

### PR DESCRIPTION
**Pull request by thanhnhung1989**
add textbox testcase

### CODE REVIEW

File: DemoQA/src/main/java/common/TestBase.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package common;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class TestBase {
+	public WebDriver driver;
```
Comment (luyht) [inline DemoQA/src/main/java/common/TestBase.java:7]: please format code

File: DemoQA/src/main/java/pages/ElementsPage.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ElementsPage extends Page {
+
```
Comment (luyht) [inline DemoQA/src/main/java/pages/ElementsPage.java:9]: Remove blank lines

**Review by luyht — COMMENTED** [review]
Please fix comments

File: DemoQA/src/main/java/common/TestBase.java (hunk 1 above)
Comment (thanhnhung1989) [reply DemoQA/src/main/java/common/TestBase.java:7]: updated

**Review by luyht — APPROVED** [review]
OK from me

**Pull request merged**